app/models/ticket_stakeholder.rb

```diff
@@ -9,6 +9,14 @@ class TicketStakeholder < ApplicationRecord
   belongs_to :ticket
   belongs_to :stakeholder, polymorphic: true
 
+  scope :belongs_to_user, lambda { |user_id|
+    where(stakeholder_id: user_id)
+  }
+
+  scope :belongs_to_groups, lambda { |group_ids|
+    where(stakeholder_type: "UserGroup").where(stakeholder_id: group_ids)
```

Comment:
Nifty Rails trick: `where` supports an unlimited number of arguments. `where(foo: 123).where(bar: nil)` can be written as `where(foo: 123, bar: nil)`

Reply:
Thanks for that, done.